src/mam/mod_mam_riak_timed_arch_yz.erl

```diff
@@ -137,12 +137,14 @@ archive_message(_Result, Host, #{message_id := MessId,
                                  local_jid := LocJID,
                                  remote_jid := RemJID,
                                  source_jid := SrcJID,
-                                 packet := Packet}) ->
+                                 packet := Packet} = Params) ->
```

Comment:
Again, I'm concerned about logging the packet 😕

Reply:
We can filter it out if needed...
exml_packet, packet and acc.